ouroboros-consensus/test-storage/Test/Ouroboros/Storage/VolatileDB/StateMachine.hs

```diff
@@ -157,6 +158,7 @@ data Cmd
     | CreateFile
     | CreateInvalidFile
     | DuplicateBlock FsPath TestBlock
+    | PutCorrupted TestBlock
```

Comment:
How is this different from `Corrupt`? To me `Corrupt` sounds more general and thus preferable.